config.yml

```
@@ -0,0 +1,6 @@
+default:
```

Suggest adding a couple of simple options here... Note, I haven't tried these to see if they throw errors
```suggestion
use_stable_links:
   google_email: ""
   chl_stable: TRUE
   chl_create_stable: FALSE
create_new_version:
   google_email: ""
   chl_stable: FALSE
   chl_create_stable: FALSE
admin_update:
```